packages/block-editor/src/components/page-template-picker/preview.native.js

```diff
@@ -59,6 +64,16 @@ const Preview = ( props ) => {
 		</View>
 	);
 
+	const rightButton = (
+		<View
+			style={ styles.headerRightButton }
+		>
+			<TouchableOpacity onPress={ onApply }>
```

Comment:
Shouldn't this be a `Button`? We can handle the exact style in the next PR, but the font and position is looking off on iOS:

![Simulator Screen Shot - iPhone 11 - 2020-01-14 at 13 09 58](https://user-images.githubusercontent.com/8739/72343535-65118a80-36cf-11ea-9a60-347cc5134211.png)

Reply:
Do you mean a React Native's `Button` ? Or the one that's already imported from  `@wordpress/components` ?

Reply:
The left button is using `Button` from `@wordpress/components`

Reply:
Now that you've made a button for the modal header, I guess I can wait for that change to be up so I can pull from it

Reply:
I agree, let's merge this one and handle the styling in https://github.com/wordpress-mobile/gutenberg-mobile/issues/1777